src/box/field_def.c

```diff
@@ -118,6 +118,7 @@ const char *field_type_strs[] = {
 	/* [FIELD_TYPE_INTERVAL] = */ "interval",
 	/* [FIELD_TYPE_ARRAY]    = */ "array",
 	/* [FIELD_TYPE_MAP]      = */ "map",
+	/* [field_type_MAX]      = */ "unknown",
```

Comment:
Please don't do that. `field_type_MAX` should never be converted to a string. You basically introduce dead code here.

Reply:
Understood. Dropped this patch.